docs/pages/resources/fundamentals-libraries.md

```diff
@@ -0,0 +1,22 @@
+---
+title: SAP Fiori Fundamentals Libraries
+keywords: libraries, frameworks
+sidebar: left-navigation-sidebar
+toc: false
+permalink: /fundamentals-libraries/
+folder: resources
```

Comment:
This page is still missing from the left hand nav - 
<img width="787" alt="Screen Shot 2019-03-25 at 7 04 58 PM" src="https://user-images.githubusercontent.com/29607818/54966254-f3decc00-4f30-11e9-9fe8-65ba37162f99.png">